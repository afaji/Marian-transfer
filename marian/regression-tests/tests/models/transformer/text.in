a Republi@@ can strategy to counter the re @-@ election of Obama
Republi@@ can leaders justified their policy by the need to combat electoral fraud .
however , the Brenn@@ an Centre considers this a my@@ th , stating that electoral fraud is r@@ arer in the United States than the number of people killed by ligh@@ tn@@ ing .
indeed , Republi@@ can lawyers identified only 300 cases of electoral fraud in the United States in a decade .
one thing is certain : these new provisions will have a negative impact on vot@@ er turn @-@ out .
in this sense , the measures will partially undermine the American democratic system .
unlike in Canada , the American States are responsible for the organisation of federal elections in the United States .
it is in this spirit that a majority of American governments have passed new laws since 2009 making the registration or voting process more difficult .
this phenomenon gained momentum following the November 2010 elections , which saw 6@@ 75 new Republi@@ can representatives added in 26 States .
as a result , 180 bil@@ ls restri@@ cting the exercise of the right to vote in 41 States were introduced in 2011 alone .
the new election laws require voters to show a photo ID card and proof of US citizenship .
furthermore , these laws also reduce early voting periods , invali@@ date the right to register as a vot@@ er on election day and withdraw the right to vote of citizens with a criminal record .
before the 2006 elections , no US State required voters to show a photo ID card .
Indi@@ ana was the first State to impose such a requirement .
in 2008 , the Supreme Court of the United States up@@ held the constitu@@ tionality of the Indi@@ ana law .
the Republi@@ can authorities were quick to extend this practice to other States .
over the past two years , they sponsored bil@@ ls in 34 States to force voters to show a photo ID card .
it is important to note that , unlike Que@@ bec , American citizens do not have a universal ID card such as the health insurance card .
in fact , 11 % of American citizens , i.e. 21 million people of voting age , do not possess a photo ID card issued by a government agency of their State .
in addition , five million new voters in 2012 do not have such identification .
and it often costs over a hundred dollars to obtain the required identity card .
the new restrictions dis@@ proportion@@ ately affect young people , minorities and people with low incomes .
in fact , 25 % of African Americans , 15 % of those ear@@ ning less than $ 35@@ ,000 ; 18 % of citizens over 65 and 20 % of voters 18 to 29 years old do not have the required photo ID card .
and that &apos;s not all .
students , voters considered to be voting more for Democratic candidates , are not allowed in several States to use the photo ID card issued by their institution .
on the other hand , these same States allow fishing or hunting club members , who vote more Republi@@ can , to use the cards issued by these clubs when they vote .
prior to 2004 , no State required proof of citizenship to vote .
Ari@@ zona was the first to introduce such a requirement .
since 2011 , a do@@ zen States have adopted laws requiring voters to prove they are American citizens .
these measures are clearly intended to limit the His@@ pan@@ ic vote .
however , it appears that two out of three His@@ pan@@ ic voters favour the Democratic party .
what is more , in 2011 Republi@@ can legislat@@ ors sponsored laws aboli@@ shing the registration of voters on election day in eight States .
in addition , they limited the right of individuals and groups to provide assistance to voters wishing to register .
these restrictions are not without consequence .
for example , during the 2004 general election , vot@@ er registration campaigns contributed to regi@@ stering around 10 million citizens .
however , the measures adopted since 2009 have led to a 17 % drop in the registration rate of new voters in 2010 compared to 2006 .
in addition , Republi@@ can legislat@@ ors have en@@ acted laws in five other States aimed at reducing the early voting period .
for example , during the 2008 general election in Florida , 33 % of early voters were African @-@ Americans , who accoun@@ ted however for only 13 % of voters in the State .
the same applied to His@@ pan@@ ics .
these represented only 11 % of voters , but 24 % of citizens who voted early .
on the other hand , 76 % of voters were white but these represented only 46 % of early voters .
of course , Democratic legislat@@ ors and their supporters vigor@@ ously opposed the adoption of laws restri@@ cting vot@@ er registration .
several bil@@ ls were blocked by v@@ eto@@ es of Democratic govern@@ ors .
the United States Att@@ or@@ ney General interven@@ ed to suspend the most controversial laws .
they were able to partially limit the damage .
for example , only 16 out of 34 States have adopted laws requiring the presentation of a photo ID card .
however , the new rules put in place will undoubtedly make it more difficult to exercise the right to vote in 2012 .
democratic critics den@@ ounce the parti@@ san character of the laws that have been passed and they see a clear objective of influ@@ encing the 2012 results in key States .
a 2011 Brenn@@ an Centre report shows that the States that have adopted these laws represent 17@@ 1 of the 2@@ 70 votes needed in the electoral college to win the Presidency .
it is too early to say with certainty that these legislative changes in the electoral system will have significant imp@@ acts on the outcome of the 2012 presidential elections .
