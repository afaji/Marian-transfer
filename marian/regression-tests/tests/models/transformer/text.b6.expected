eine republi@@ kanische Strategie gegen die Wieder@@ wahl Ob@@ amas
die republi@@ kanische Führung recht@@ fertigte ihre Politik durch die Notwendigkeit , Wahl@@ betrug zu bekämpfen .
das Brenn@@ an @-@ Zentrum betrachtet dies jedoch als einen My@@ thos , der besagt , dass Wahl@@ betrug in den Vereinigten Staaten sel@@ tener ist als die Zahl der durch Bli@@ tz getö@@ teten Menschen .
tatsächlich haben republi@@ kanische An@@ wälte innerhalb eines Jahrzeh@@ n@@ ts nur 300 Fälle von Wahl@@ betrug in den Vereinigten Staaten festgestellt .
eines ist sicher : diese neuen Bestimmungen werden sich negativ auf die Wahl@@ beteiligung auswirken .
in diesem Sinne werden die Maßnahmen das amerikanische demokratische System teilweise untergraben .
anders als in Kanada sind die Vereinigten Staaten für die Organisation von Bun@@ dest@@ ag@@ s@@ wahlen in den Vereinigten Staaten verantwortlich .
in diesem Sinne hat eine Mehrheit der amerikanischen Regierungen seit 2009 neue Gesetze erlassen , die die Registrierung oder das Abstimm@@ ungsverfahren ersch@@ weren .
dieses Phänomen gewann an Dynamik nach den Wahlen im November 2010 , bei denen 6@@ 75 neue republi@@ kanische Vertreter in 26 Staaten hinzu@@ kamen .
infolgedessen wurden allein 2011 180 Gesetze eingeführt , die die Ausübung des Wahl@@ rechts in 41 Staaten einschränken .
die neuen Wahl@@ gesetze verlangen von den Wählern , dass sie einen Licht@@ bild@@ aus@@ weis und einen Beweis der US @-@ Staats@@ bürgerschaft vorlegen .
darüber hinaus reduzieren diese Gesetze auch die vorge@@ zo@@ genen Wahl@@ fri@@ sten , berau@@ ben das Recht , sich am Wahl@@ tag als Wähler zu registrieren und ziehen das Wahl@@ recht für Bürger mit einem Strafre@@ gi@@ ster zurück .
vor den Wahlen 2006 ver@@ lang@@ te kein US @-@ Staat , dass die Wähler einen Licht@@ bild@@ aus@@ weis vorlegen müssten .
Indi@@ ana war der erste Staat , der eine solche Forderung stellte .
2008 hat der Ober@@ ste Gerichtshof der Vereinigten Staaten die Verfassungs@@ mäßigkeit des Indi@@ ana @-@ Gesetzes bestätigt .
die republi@@ kanischen Behörden waren schnell dabei , diese Praxis auf andere Staaten auszu@@ dehnen .
in den letzten zwei Jahren haben sie in 34 Staaten Rechn@@ ungen gesp@@ on@@ sert , um die Wähler zu zwingen , einen Licht@@ bild@@ aus@@ weis vorzulegen .
es ist wichtig festzustellen , dass amerikanische Bürger im Gegensatz zu Que@@ bec keinen allgemeinen Aus@@ weis wie die Kranken@@ versicherungs@@ karte haben .
tatsächlich verfügen 11 % der amerikanischen Bürger , d. h. 21 Millionen Menschen im Wahl@@ alter , nicht über einen Aus@@ weis@@ aus@@ weis , den eine staatliche Behörde ihres Staates ausgestellt hat .
darüber hinaus verfügen 2012 fünf Millionen neue Wähler nicht über eine solche Identi@@ fikation .
und es kostet oft über hundert Dollar , den erforderlichen Personal@@ aus@@ weis zu erhalten .
die neuen Beschränkungen betreffen über@@ proportional junge Menschen , Minderheiten und Menschen mit niedri@@ gem Einkommen .
tatsächlich haben 25 % der afrikanischen Amerikaner , 15 % derjenigen , die weniger als 3@@ 5.000 Dollar verdienen ; 18 % der Bürger über 65 und 20 % der Wähler über 18 bis 29 Jahre nicht die erforderliche Aus@@ weis@@ karte .
und das ist noch nicht alles .
Studenten , Wähler , die mehr für demokratische Kandidaten stimmen , dürfen in mehreren Staaten nicht den von ihrer Institution ausge@@ stellten Licht@@ bild@@ aus@@ weis benutzen .
andererseits erlauben dieselben Staaten den Mitgliedern des Fischerei@@ - oder Jag@@ d@@ klu@@ bs , die mehr Republi@@ kaner wählen , die von diesen Clubs bei der Abstimmung ausge@@ stellten Karten zu verwenden .
vor 2004 ver@@ lang@@ te kein Staat einen Nachweis der Staats@@ bürgerschaft , um abzustimmen .
Ari@@ zona war die erste , die eine solche Anforderung eingeführt hat .
seit 2011 haben ein Dutz@@ end Staaten Gesetze verabschiedet , die die Wähler verpflichten , zu beweisen , dass sie amerikanische Bürger sind .
diese Maßnahmen zielen eindeutig darauf ab , die Abstimmung in His@@ pan@@ ia einzu@@ schränken .
es scheint jedoch , dass zwei von drei h@@ is@@ pan@@ ischen Wählern die Demokrati@@ sche Partei befürworten .
darüber hinaus unterstützten die republi@@ kanischen Gesetzgeber 2011 Gesetze , die die Registrierung von Wählern am Wahl@@ tag in acht Staaten ab@@ schaff@@ ten .
außerdem schrän@@ kten sie das Recht von Einzelpersonen und Gruppen ein , Wähler@@ innen und Wähler zu unterstützen , die sich registrieren möchten .
diese Beschränkungen sind nicht ohne Folgen .
so tru@@ gen Wahl@@ registri@@ erungs@@ kamp@@ agnen während der Parlaments@@ wahlen 2004 zur Registrierung von etwa 10 Millionen Bürgern bei .
die seit 2009 er@@ gri@@ ffenen Maßnahmen haben jedoch zu einem Rückgang der Zul@@ assungs@@ rate für neue Wähler um 17 % im Jahr 2010 im Vergleich zu 2006 geführt .
darüber hinaus haben republi@@ kanische Gesetzgeber in fünf anderen Staaten Gesetze erlassen , die darauf abzielen , die vorge@@ zo@@ gene Wahl@@ periode zu ver@@ kürzen .
so waren beispielsweise während der Parlaments@@ wahlen 2008 in Florida 33 % der frühen Wähler A@@ fro@@ amerikan@@ er , auf die jedoch nur 13 % der Wähler im Staat ent@@ fielen .
dasselbe galt für die His@@ pan@@ ics .
diese re@@ präsen@@ tierten nur 11 % der Wähler , aber 24 % der Bürger , die früh gewählt haben .
andererseits waren 76 % der Wähler weiß , aber diese stellten nur 46 % der frühen Wähler dar .
natürlich haben sich die demokratischen Gesetzgeber und ihre Anhänger v@@ eh@@ ement gegen die Verabschiedung von Gesetzen ausgesprochen , die die Registrierung der Wähler einschränken .
mehrere Gesetze wurden durch V@@ et@@ os demokratischer Gouverne@@ ure blockiert .
der General@@ staatsan@@ walt der Vereinigten Staaten hat interven@@ iert , um die umstrit@@ ten@@ sten Gesetze aus@@ zusetzen .
Sie konnten den Schaden teilweise begrenzen .
so haben beispielsweise nur 16 von 34 Staaten Gesetze verabschiedet , die die Vorlage eines Foto@@ aus@@ we@@ ises vorsch@@ reiben .
die neuen Regeln werden es jedoch zweifellos schwieriger machen , 2012 das Wahl@@ recht auszuüben .
demokratische Kritiker verurteilen den partei@@ ischen Charakter der verabschiedeten Gesetze und sehen ein klares Ziel , die Ergebnisse von 2012 in Schlüssel@@ staaten zu beeinflussen .
ein Bericht des Brenn@@ an Centre aus dem Jahr 2011 zeigt , dass die Staaten , die diese Gesetze verabschiedet haben , 17@@ 1 der 2@@ 70 Stimmen ausmachen , die im Wahl@@ gre@@ mium benötigt werden , um den Ratsvorsitz zu gewinnen .
es ist zu früh , mit Sicherheit zu sagen , dass diese Gesetzes@@ änderungen im Wahl@@ system erhebliche Auswirkungen auf das Ergebnis der Präsidentschaftswahlen 2012 haben werden .
